CHANGELOG.rst

```diff
@@ -27,8 +27,10 @@ Scripting improvements
 - ``fish_indent`` allows to write inline variable assignments on multiple lines (ending in a backslash), instead of joining them into one line (:issue:`7955`).
 - fish gained a ``--no-config`` option to disable reading the configuration. This applies both to the user's and the admin's config.fish (typically in /etc/fish/config.fish) and snippets and also sets $fish_function_path to just the functions shipped with fish and disables universal variables and history. (:issue:`7921`, :issue:`1256`).
 - When universal variables are unavailable for some reason, setting a universal variable now sets a global variable instead (:issue:`7921`).
+- ``$last_pid`` now reports the pid of the last process in the pipeline, allowing it to be used in scripts.
 - ``process-exit`` event handlers now receive the same value as ``$status`` in all cases, instead of receiving -1 when the exit was due to a signal.
 - ``process-exit`` event handlers for pid 0 also received ``JOB_EXIT`` events; this has been fixed.
+- ``job-exit`` event handlers may be created with any of the pids from the job, instead of just the process group.
```

Comment:
So the process group here still works because it's one of the PIDs, not because we also specifically look at the PGID? Do we then still need to mention the PGID at all?

Reply:
Oh, also the reverted change should have the CHANGELOG entry removed.

Reply:
Right. Good point, I deleted the mention of the process group. Also reverted the other changelog entry.